roles/multi-tenancy/tasks/tenant-cas-customizations.yaml

```diff
@@ -0,0 +1,79 @@
+---
+- name: update tenant cas customizations
+  block:
+    - name: tenant cas - MPP workers
```

Comment:
Are these truly MPP workers? Just want to verify or if that's the only way to have multi-tenancy support.

Reply:
This task is to set the tenant CAS to MPP when the worker_count is specified. Same as provider, it's the MPP support for tenants.

Reply:
Ok, where's the logic with the check for `worker_count`? Not seeing it.

Reply:
See line number 8.
From the branch: https://github.com/sassoftware/viya4-deployment/blob/cas_updates/roles/multi-tenancy/tasks/tenant-cas-customizations.yaml#L8

Reply:
thx.